### ISSUE TITLE
Social Activity Feed Click-through

### ISSUE THREAD

**Issue by jterps08** [PowerlineApp/powerline-mobile#303]
![image](https://cloud.githubusercontent.com/assets/7582457/18898876/8aed2570-8502-11e6-8d8d-a82fdaff04a5.png)

The first item in the SAF above - "Peter10 commented on a post you subscribed to"- I am unclear as to why I received a push notification for this. I cannot determine which post _exactly_ that this update is referring to because clicking on it does not bring me to the post, it brings me to the newsfeed.... I am not sure if 1) I am being subscribed to some post by accident (bug) or if 2) Peter10 made a comment on something I subscribed to a while ago but I can't identify because I can't click to navigate directly to it.

We need to solve either 1 or 2 as part of this issue. I want to make sure I actually subscribed to the post  and that the system is not accidentally subscribing me to a post.

**Comment by igaponov** [comment]
https://github.com/PowerlineApp/powerline-mobile/issues/298

**Comment by petervojtek** [comment]
@jterps08 , this was a bug and I fixed it:

>  I cannot determine which post exactly that this update is referring to because clicking on it does not bring me to the post, it brings me to the newsfeed

Now when you click it you will be navigated to the post/user-petition detail.

**But in general my problem is that I don't know which behaviour should trigger new notification in SAF (Following tab) and which not**. For example, I tried did this:
1. As user Peter31 I subscribed to poll:crowdfunding 295
2. As User Peter10 I added a comment to poll:crowdfunding 295
3. I expected to receive a SAF notification on new comment as Peter31, but none arrived. 

This means to be able to test the changes Igor made (adjusting which social items to belong to Following tab and which not) I would need some specification..
